dbt/include/trino/macros/materializations/incremental.sql

```diff
@@ -1,6 +1,33 @@
-{% macro dbt_trino_get_append_sql(tmp_relation, target_relation) %}
+{% macro dbt_trino_validate_get_incremental_strategy(config) %}
+  {#-- Find and validate the incremental strategy #}
+  {%- set strategy = config.get("incremental_strategy", default="append") -%}
+
+  {% set invalid_strategy_msg -%}
+    Invalid incremental strategy provided: {{ strategy }}
+    Expected one of: 'append', 'delete+insert'
+  {%- endset %}
+  {% if strategy not in ['append', 'delete+insert'] %}
+    {% do exceptions.raise_compiler_error(invalid_strategy_msg) %}
+  {% endif %}
+
+  {% do return(strategy) %}
+{% endmacro %}
+
+{% macro dbt_trino_get_incremental_sql(strategy, tmp_relation, target_relation, unique_key, dest_columns) %}
```

Comment:
is there any specific reason why the new methods are prefixed with "dbt_trino_" ?

Reply:
I based myself on the snowflake implementation. I think their reasoning is that macro's are global and prefixing it will avoid name clashes from happening, similar like java namespaces.

See [snowflake adapter](https://github.com/dbt-labs/dbt-snowflake/blob/main/dbt/include/snowflake/macros/materializations/incremental.sql)